lib/ood_core/job/adapters/helper.rb

```diff
@@ -26,7 +26,7 @@ def self.ssh_wrap(submit_host, cmd, cmd_args, strict_host_checking = true, env =
           return cmd, cmd_args if submit_host.to_s.empty?
 
           check_host = strict_host_checking ? "yes" : "no"
-          args = ['-o', 'BatchMode=yes', '-o', 'UserKnownHostsFile=/dev/null', '-o', "StrictHostKeyChecking=#{check_host}", "#{submit_host}"]
+          args = ['-p', ENV["OOD_SSH_PORT"].nil? ? "22" : "#{ENV['OOD_SSH_PORT']}", '-o', 'BatchMode=yes', '-o', 'UserKnownHostsFile=/dev/null', '-o', "StrictHostKeyChecking=#{check_host}", "#{submit_host}"]
```

Comment:
Can you put this logic in a helper function like this so other things can just refercen the fuction without the logic. (also be sure to cast to an integer)

```ruby
def ssh_port
  ENV["OOD_SSH_PORT"].nil? ? "22" : ENV['OOD_SSH_PORT'].to_i.to_s
end

```

Reply:
> Can you put this logic in a helper function like this so other things can just refercen the fuction without the logic. (also be sure to cast to an integer)
> 
> ```ruby
> def ssh_port
>   ENV["OOD_SSH_PORT"].nil? ? "22" : ENV['OOD_SSH_PORT'].to_i.to_s
> end
> ```

The Helper is a good idea.  Thanks!